src/Components/Projects/index.js

```diff
@@ -27,6 +27,24 @@ const Projects = props => {
 
     item.display = true;
 
+    const LabelsCollection = new Collection(
+      props.firebase,
+      `/projects/${ docRef.id }/labels`
+    );
+
+    const defaultLabel = {
+      label: 'Default',
+      color: 'yellow',
+      value: 'yellow',
+      description: ''
+    };
+
+    const labelDocRef = await LabelsCollection.postItem(defaultLabel);
+
+    labelDocRef.update({
+      id: labelDocRef.id
+    });
+
```

Comment:
```suggestion
    const defaultLabel = {
      label: 'Default',
      color: 'yellow',
      value: 'yellow',
      description: ''
    };

    const labelDocRef = await LabelsCollection.postItem(defaultLabel);

    labelDocRef.update({
      id: labelDocRef.id
    });
createDefaultLabel()
```

and move out the logic to a function called `createDefaultLabel()`

Functionally not different, but keeping functions small and self-explanatory :)